Composer/packages/server/src/models/bot/__tests__/botProject.test.ts

```diff
@@ -44,7 +44,7 @@ beforeEach(async () => {
 describe('init', () => {
   it('should get project successfully', () => {
     const project: { [key: string]: any } = proj.getProject();
-    expect(project.files.length).toBe(15);
+    expect(project.files.length).toBe(16);
```

Comment:
I don't understand these changes, can you elaborate?

Reply:
We pick up JSON files now, so this added appsettings.json to the project files list.  The JSON files are filters to skills manifests and JSON schemas in the indexers so it won't hurt anything.